core/runtime/src/main/java/io/quarkus/runtime/logging/LoggingSetupRecorder.java

```diff
@@ -151,7 +153,8 @@ private static boolean hasColorSupport() {
 
     private static Handler configureConsoleHandler(final ConsoleConfig config, final ErrorManager defaultErrorManager,
             final List<LogCleanupFilterElement> filterElements,
-            final List<RuntimeValue<Optional<Formatter>>> possibleFormatters) {
+            final List<RuntimeValue<Optional<Formatter>>> possibleFormatters,
+            final Supplier<String> bannerSupplier) {
```

Comment:
I have added Banner Supplier which itself contains Banner as String. As per @dmlloyd  I should integrate banner here. How to integrate the supplier string to formatter here ?

Reply:
Do I have to create a custom formatter that returns the banner when `formatter.getHead(Handler)` is called ? or make changes to PatternFormatter?